src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
-body {
+$font-family: "Roboto", arial, sans-serif;
+$text-color-main: #060b35;
+$text-color-secondary: #616070;
+$button-color: #00acdc;
+$margin-bottom: 16px;
+
+*,
+*::after,
+*::before {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: $font-family;
+}
+
+.product-card {
```

Comment:
according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) for each BEM block we should have 1 file - please create 1 file for card and 1 for stars, and move their piece of code